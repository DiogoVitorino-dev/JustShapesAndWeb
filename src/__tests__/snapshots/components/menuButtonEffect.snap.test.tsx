import { render } from "@testing-library/react-native";

import MenuButtonEffect from "@/components/menu/menuButtonEffect";

describe("Menu Button Effect - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<MenuButtonEffect />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
