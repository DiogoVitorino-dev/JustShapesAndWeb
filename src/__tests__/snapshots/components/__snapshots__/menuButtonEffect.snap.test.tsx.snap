// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu Button Effect - Snapshot component test should render correctly 1`] = `
<RNSVGSvgView
  bbHeight="100%"
  bbWidth="90%"
  focusable={false}
  height="100%"
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": "100%",
        "width": "90%",
      },
    ]
  }
  width="90%"
>
  <RNSVGGroup
    fill={
      {
        "payload": 4278190080,
        "type": 0,
      }
    }
  >
    <RNSVGRect
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      height="100%"
      propList={
        [
          "fill",
          "strokeWidth",
        ]
      }
      strokeWidth="1"
      width="50%"
      x="0"
      y="0"
    />
    <RNSVGRect
      fill={
        {
          "payload": 4278190080,
          "type": 0,
        }
      }
      height="100%"
      matrix={
        [
          1,
          0,
          0.4663076581549986,
          1,
          0,
          0,
        ]
      }
      propList={
        [
          "fill",
        ]
      }
      width="50%"
      x="0"
      y="0"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
